src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -178,7 +179,8 @@
 short-summary: List network rules.
 examples:
   - name: List network rules. (autogenerated)
-    text: az storage account network-rule list --account-name MyAccount --resource-group MyResourceGroup
+    text: |
+        az storage account network-rule list --account-name MyAccount --resource-group MyResourceGroup
```

Comment:
prefer lowercase() for storage account name.

Reply:
Should it be better to udpate all the --account-name? Otherwise Aladdin will generated `MyAccount` next release.

Reply:
sure. But for the PR, please help change MyAccount to myaccount. Thanks